models/electric_relaxation/configs/config_model.py

```diff
@@ -7,7 +9,7 @@ def get_model_config():
     """
     model_config = {
         "name": "electric_relaxation",
-        "algorithm": "RandomForestClassifier", 
+        "algorithm": RandomForestClassifier, 
         "depvar": "ged_sb_dep", #or target? 
```

Comment:
I know this is inherited but I think "depvar" should be "target". Also, see notes and thoughts on the configs in Slack. In short I think (as you also suggested today) that this script should be called config_meta.

Reply:
Again, you can look at purple alien if need be